Release DAML SDK 0.13.47 (#4258)
digital-asset/daml: docs/source/support/release-notes.rst

This page contains release notes for the SDK.

.. _release-0-13-47:

0.13.47 - 2020-01-29
--------------------

DAML Compiler
~~~~~~~~~~~~~

- ``damlc test`` now initializes the packagedb automatically which means that
  it will work on projects that declare custom ``dependencies`` in
  ``daml.yaml`` without having to call ``damlc init`` first.
- Choices marked explicitly as ``preconsuming`` are now equivalent to a
  ``nonconsuming`` choice that calls ``archive self`` at the beginning.

DAML Integration Kit
~~~~~~~~~~~~~~~~~~~~

- The simplified kvutils API now uses ``com.digitalasset.resources`` to manage
  acquiring and releasing resources instead of ``Closeable``.

DAML Standard Library
~~~~~~~~~~~~~~~~~~~~~

- Add ``CanAbort`` instance for ``Either Text``.

DAML Studio
~~~~~~~~~~~

> - Allow all command line flags for ``daml build`` for DAML Studio as well.

Reviewer: ```suggestion
- All ``build-options`` supported by ``daml build`` are now also supported in DAML studio.
```

I don’t think users really understand or should understand what command line flags for `DAML Studio` are.

Author: I totally agree. Thanks.
